lib/server/index.js

```diff
@@ -52,6 +52,25 @@ class GameServer {
     process.stdout.write(`Connection from ${socket.conn.remoteAddress}\n`);
   }
 
+  listGames() {
```

Comment:
You need to list games both from the API (GET) and the socket from the browser. The iteration is the same in both cases so having it here is DRY.